### PR TITLE
Upgrade to ES/Kibana version v7.9.1

### PR DESCRIPTION
**Pull request by zhongnansu**
*Issue #, if available:*

*Description of changes:*
Upgrade to ES/Kibana version v7.9.1 for both reports-scheduler plugin and kibana-reports plugin
- Kibana reports plugin
  - server
    - replace `IClusterClient` ,`IScopedClusterClient` to`ILegacyClusterClient`, `ILegacyScopedClusterClient`
    - replace `...dataClient.callAsCurrentUser` to `...legacy.client.callAsCurrentUser`
  - UI 
    - add `data` to `kibana.json` -> `requiredPlugins`
- Reports scheduler
  - bump version/dependency number

- This update also solves one of the kibana plugin build issue in #40. The issue that  `kibana-plugin-helper` didn't pick up hidden files started with dot, which in our case is the `.local-chromium`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by akbhatta — APPROVED** [review]

File: kibana-reports/server/plugin.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import {
   CoreStart,
   Plugin,
   Logger,
-  IClusterClient,
+  ILegacyClusterClient,
```
Comment (davidcui-amzn) [inline kibana-reports/server/plugin.ts:22]: We revert back to legacy for 7.9?

**Review by davidcui-amzn — COMMENTED** [review]
Does the UI still render with the version change when navigating to the Reporting homepage?

File: kibana-reports/server/plugin.ts (hunk 1 above)
Comment (zhongnansu) [reply kibana-reports/server/plugin.ts:22]: yes, the `IClusterClient` in kibana v7.9.1 is not fully supported for our usage.

**Comment by zhongnansu** [comment]
> Does the UI still render with the version change when navigating to the Reporting homepage?

The only issues that UI is complaining is the now it requires a `data` plugins in the kibana.json. Everything looks good after I added that

File: kibana-reports/server/plugin.ts (hunk 1 above)
Comment (davidcui-amzn) [reply kibana-reports/server/plugin.ts:22]: Got it

**Review by davidcui-amzn — APPROVED** [review]
LGTM

**Pull request merged**